specification/types.md

```diff
@@ -50,6 +51,7 @@ A structure which contains a subset of the fields defined in the `evaluation det
 - error message (string, optional)
 - reason (string, optional)
 - variant (string, optional)
+- flag metadata ([flag metadata](#flag-metadata), optional)
```

Comment:
@toddbaert note that this new field would be optional in Resolution Details but required in Evaluation Details - the SDK would be responsible for defaulting to an empty record  in the Evaluation Details if the provider didn't provide anything int he Resolution Details. 

I'm not sure how much of a wrinkle that will create in the typing for the SDKs; I think some of them likely implement Evaluation Details as a subtype of Resolution Details, and this change would break that I suspect.

Reply:
Yep, I don't think that's a problem.